### PR TITLE
feat(): create icon file on ssc init

### PR DESCRIPTION
**Pull request by chicoxyzzy**

### CODE REVIEW

File: src/cli/templates.hh [hunk 1]
```diff
@@ -996,7 +996,7 @@ constexpr auto gXCodePlist = R"XML(<?xml version="1.0" encoding="UTF-8"?>
   <key>CFBundleIdentifier</key>
   <string>{{meta_bundle_identifier}}</string>
   <key>CFBundleIconFile</key>
-  <string>ui/icon.png</string>
+  <string>{{mac_icon}}</string>
```
Comment (jwerle) [inline src/cli/templates.hh:999]: this should be `{{ios_icon}}`

**Review by jwerle — CHANGES_REQUESTED** [review]

File: src/cli/templates.hh (hunk 1 above)
Comment (chicoxyzzy) [inline src/cli/templates.hh:999]: is this correct? @heapwolf

File: src/cli/templates.hh [hunk 2]
```diff
@@ -1659,10 +1659,7 @@ headers = native-module1.hh
 ; cmd = "node backend/index.js"
 
 ; The icon to use for identifying your app on Windows.
-icon = ""
-
-; The icon to use for identifying your app on Windows.
-logo = "src/icons/icon.png"
```
Comment (chicoxyzzy) [inline src/cli/templates.hh:1665]: we don't use it, seems like legacy code

File: src/cli/templates.hh [hunk 3]
```diff
@@ -1624,17 +1624,14 @@ icon = "src/icon.png"
 
 [mac]
 
-; Mac App Store icon
-appstore_icon = "src/icons/icon.png"
-
```
Comment (chicoxyzzy) [inline src/cli/templates.hh:1629]: not used

File: src/cli/templates.hh [hunk 4]
```diff
@@ -1659,10 +1656,7 @@ headers = native-module1.hh
 ; cmd = "node backend/index.js"
 
 ; The icon to use for identifying your app on Windows.
-icon = ""
-
-; The icon to use for identifying your app on Windows.
-logo = "src/icons/icon.png"
```
Comment (chicoxyzzy) [inline src/cli/templates.hh:1665]: legacy? changed all `win_logo` to be `win_icon`

Comment (heapwolf) [reply src/cli/templates.hh:1665]: win_logo was for the app package manifest, win_icon is the actual icon of the app, there may be a better name for it

Comment (chicoxyzzy) [reply src/cli/templates.hh:1665]: I brought `win_logo` back to not block this PR, we can change it's name later if necessary

Comment (chicoxyzzy) [reply src/cli/templates.hh:1665]: `[mac] appstore_icon` is not used anywhere so I removed it until we need it

**Review by jwerle — APPROVED** [review]

**Pull request merged**